heudiconv/heuristics/reproin.py

```diff
@@ -541,8 +540,14 @@ def infotodict(seqinfo):
 
         # analyze s.protocol_name (series_id is based on it) for full name mapping etc
         if seqtype == 'func' and not seqtype_label:
-            if '_pace_' in series_spec:
+            if s.series_description.endswith('_SBRef'):
+                seqtype_label = 'sbref'
+            elif '_pace_' in series_spec:
                 seqtype_label = 'pace'  # or should it be part of seq-
+            elif 'P' in s.image_type:
+                seqtype_label = 'phase'
```

Comment:
this is already under `seqtype == 'func'` so if I read the code correctly, it wouldn't work for a simple `fmap`

Reply:
I think a separate if statement needs to be added for `seqtype == 'fmap'` to check `if 'P' in s.image_type`. Does that sound right?

Reply:
Wait, sorry, if `seqtype` is already known to be `fmap` then there's no reason to check `s.image_type`. I don't know if it's a problem though, since there's definitely a step that determines `seqtype_label` for field maps that's separate from this part. So this if statement should only apply to functional data.